dev/source/docs/building-ardupilot-onwindows10.rst

```diff
@@ -1,29 +1,24 @@
 
-.. _building-ardupilot-onwindows10:
+.. _building-ardupilot-on-windows10:
```

Comment:
You can't just change the reference name without changing it where it is linked.